classes/blocks/class-cookies.php

```diff
@@ -24,13 +24,12 @@ public function __construct() {
 		// - Register the block for the editor
 		// in the PHP side.
 		register_block_type(
-			'planet4-blocks/cookies',
+			self::BLOCK_NAMESPACE_PREFIX . '/' . self::BLOCK_NAME,
 			[
-				'editor_script'   => 'planet4-blocks/cookies',
 				// todo: Remove when all content is migrated.
 				'render_callback' => static function ( $attributes ) {
 					$json = wp_json_encode( [ 'attributes' => $attributes ] );
-					return '<div data-render="planet4-blocks/cookies" data-attributes="' . htmlspecialchars( $json ) . '"></div>';
+					return '<div data-render="' . self::BLOCK_NAMESPACE_PREFIX . '/' . self::BLOCK_NAME . '" data-attributes="' . htmlspecialchars( $json ) . '"></div>';
```

Comment:
This is also the same for all front end rendered blocks. So instead of replacing the name with the constants everywhere, we could extract it to the base block class as well.

Reply:
I added the function, I didn't extract the render callback to `Base_Block` in case we want to do some custom logic here, as happens with the attributes for some blocks.

Reply:
I was more thinking of putting just what is now returned into a function, not the entire `render_callback`. But either way works for me :ok_hand: